### PR TITLE
fix: Set proper values for titles and descriptions

### PR DESCRIPTION
**Pull request by aramovic79**
### Addressing

**groups** 

- [x] `"title": "Catalog Service”` → i.e. without appending “ Title”

**packages**

- [x] `"title": "capire bookshop”` → take what is there, don’t add extra text like “sample title for"
- [x] `“shortDescription"` → take what is there, don’t add extra text like “Here’s the short description for ..."
- [x] `“description"` → take what is there, don’t add extra text like “Here’s the short description for ..."

**apiResources**

- [x] `"title": "Catalog Service”` → take what is there, don’t add extra text like “The service is for ..."
- [x] `“shortDescription": "Catalog Service” `→ take what is there, don’t add extra text like “Here we have the shortDescription for ..."
- [x] `“description": "Catalog Service”` → take what is there, don’t add extra text like “Here we have the description for ..."

from https://github.com/cap-js/ord/issues/38

### CODE REVIEW

**Comment by aramovic79** [comment]
cc: @ductaily @Fannon

**Review by RamIndia — APPROVED** [review]
Looks Good

File: lib/templates.js [hunk 1]
```diff
@@ -252,22 +268,35 @@ const fCreateAPIResourceTemplate = (srv, srvDefinition, global,packageIds) => {
...
-                  srvDefinition['@description'] ?? srvDefinition['@Core.Description'] ??
-                  `This is an example event catalog that contains only a partial ODM ${global.appName} V1 event`,
-    version:  ordExtensions.version ?? "1.0.0",
-    releaseStatus:  ordExtensions.releaseStatus ?? "beta",
-    partOfPackage: _getPackageID(global.capNamespace,packageIds,'event'),
+    description:
+      ordExtensions.description ??
+      srvDefinition["@description"] ??
+      srvDefinition["@Core.Description"] ??
+      `This is an example event catalog that contains only a partial ODM ${global.appName} V1 event`,
```
Comment (Fannon) [inline lib/templates.js:296]: Is this some leftover from a copied example? 

```suggestion
      `CAP Event resource which may include a collection of events / messages.`,
```

**Comment by RamIndia** [comment]
@aramovic79 : You can resolve the conflicts and merge this PR.

**Pull request merged**